src/main/resources/application.yml

```diff
@@ -0,0 +1,12 @@
+server:
+    port: 9000
+logging:
+    level:
+        org.springframework.security: trace
+
+
+oauth2client:
+    oidcClientId: "oidc-client"
+    oidcClientSecret: "{noop}secret"
```

Comment:
Actually, we need to encrypt the client secret using something like the Bcrypt algorithm. 
However, it seems we will change the client storage into the database, so it's still fine for this MR.